openmc/mgxs/__init__.py

```diff
@@ -1,4 +1,43 @@
+
 from openmc.mgxs.groups import EnergyGroups
 from openmc.mgxs.library import Library
 from openmc.mgxs.mgxs import *
 from openmc.mgxs.mdgxs import *
+
+GROUP_STRUCTURES = {}
+"""Dictionary of commonly used energy group structures, including "CASMO-X" (where X
+is 2, 4, 8, 16, 25, 40 or 70) from the CASMO_ lattice physics code.
+
+.. _CASMO: https://www.studsvik.com/SharepointFiles/CASMO-5%20Development%20and%20Applications.pdf
+
+"""
+
+GROUP_STRUCTURES['CASMO-2'] = np.array([
```

Comment:
One more thing before we merge: add `import numpy as np` at the top of this module to not rely on transitive imports.

Reply:
Done